app/utils/custom-nunjucks-filters/govuk-date.js

```diff
@@ -0,0 +1,27 @@
+const { DateTime } = require('luxon')
```

Comment:
Where is this file being used?

Reply:
see https://github.com/alphagov/pay-selfservice/pull/4323/files#diff-6c6729ab30442e4e21ba0ca2f077494c1b57e7814c23ea9f9e4e50beac36149eR48